[
  {
    "class": "django.core.exceptions.ImproperlyConfigured",
    "message": "You're using the Django \"sites framework\" without having set the SITE_ID setting. Create a site in your database and set the SITE_ID setting or pass a request to Site.objects.get_current() to fix this error.",
    "stack": [
      "src/oscar/apps/communication/utils.py:169",
      "src/oscar/apps/communication/utils.py:176",
      "src/oscar/apps/customer/utils.py:32"
    ]
  },
  {
    "class": "django.core.exceptions.ImproperlyConfigured",
    "message": "You're using the Django \"sites framework\" without having set the SITE_ID setting. Create a site in your database and set the SITE_ID setting or pass a request to Site.objects.get_current() to fix this error.",
    "stack": [
      "src/oscar/apps/communication/utils.py:176",
      "src/oscar/apps/customer/utils.py:32"
    ]
  },
  {
    "class": "django.core.exceptions.ImproperlyConfigured",
    "message": "You're using the Django \"sites framework\" without having set the SITE_ID setting. Create a site in your database and set the SITE_ID setting or pass a request to Site.objects.get_current() to fix this error.",
    "stack": [
      "src/oscar/apps/customer/utils.py:32"
    ]
  }
]